modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories/snout.dm

```diff
@@ -433,3 +433,27 @@
 	color_src = USE_MATRIXED_COLORS
 	name = "Acrador (Short)"
 	icon_state = "acrador_short"
+
+/datum/sprite_accessory/snouts/acrador_1
+	icon = 'modular_skyrat/master_files/icons/mob/sprite_accessory/snouts.dmi'
+	color_src = USE_MATRIXED_COLORS
+	name = "Acrador 1 (Normal)"
+	icon_state = "acrador_1"
+
+/datum/sprite_accessory/snouts/acrador_2
+	icon = 'modular_skyrat/master_files/icons/mob/sprite_accessory/snouts.dmi'
+	color_src = USE_MATRIXED_COLORS
+	name = "Acrador 2 (Normal)"
+	icon_state = "acrador_2"
+
+/datum/sprite_accessory/snouts/acrador_3
+	icon = 'modular_skyrat/master_files/icons/mob/sprite_accessory/snouts.dmi'
+	color_src = USE_MATRIXED_COLORS
+	name = "Acrador 3 (Normal)"
+	icon_state = "acrador_3"
+
+/datum/sprite_accessory/snouts/acrador_4
+	icon = 'modular_skyrat/master_files/icons/mob/sprite_accessory/snouts.dmi'
+	color_src = USE_MATRIXED_COLORS
+	name = "Acrador 4 (Normal)"
+	icon_state = "acrador_4"
```

Comment:
Could you give them names that aren't just 1 to 4, to better represent what they're meant to be?

Reply:
I've asked a person who requested this addition and they told me that there's no specific names for them, like regional differences, except the pattern sometimes being specific to certain colors.